### PR TITLE
Update postresql wallet plugin depedencies

### PR DESCRIPTION
**Pull request by Patrik-Stas**
- Enhance logging on failure (the information about why initialization or creation of storage failed was previously lost).
- Update cargo dependencies on postgresql storage 
- Remove ursa 0.1.1 as optional dependency - After update, I couldn't get ursa 0.1.1 t build due to some trait implementations problem against rng library. It's however also not clear to me why do we need ursa? (It works without ursa dependency as well) But I might be missing something here.

### CODE REVIEW

**Comment by Patrik-Stas** [comment]
Hi @ianco can you take a look?

File: experimental/plugins/postgres_storage/src/lib.rs [hunk 1]
```diff
@@ -49,13 +48,19 @@ use std::collections::HashMap;
...
 pub static POSTGRES_STORAGE_NAME: &str = "postgres_storage";
 
 
 #[no_mangle]
 pub extern fn postgresstorage_init() -> libindy::ErrorCode {
     let postgres_storage_name = CString::new(POSTGRES_STORAGE_NAME).unwrap();
+    let res: Result<(), CommonError> = LibindyDefaultLogger::init(None).into();
+    match res {
+        _ => {}
+    }
```
Comment (Artemkaaas) [inline experimental/plugins/postgres_storage/src/lib.rs]: you can use `.ok()` instead it

**Comment by ianco** [comment]
Looks good to me (other than @Artemkaaas feedback)

File: experimental/plugins/postgres_storage/src/lib.rs [hunk 2]
```diff
@@ -49,13 +48,19 @@ use std::collections::HashMap;
...
+use utils::logger::LibindyDefaultLogger;
+use errors::common::CommonError;
 
 pub static POSTGRES_STORAGE_NAME: &str = "postgres_storage";
 
 
 #[no_mangle]
 pub extern fn postgresstorage_init() -> libindy::ErrorCode {
     let postgres_storage_name = CString::new(POSTGRES_STORAGE_NAME).unwrap();
+    let res: Result<(), CommonError> = LibindyDefaultLogger::init(None).into();
```
[the remark this answers is not in the document]
Comment (Patrik-Stas) [reply experimental/plugins/postgres_storage/src/lib.rs]: Hmmm, sorry I am not sure what's the required change. The link you've posted is implementation of nullpay logger, the initialization of null pay logger is as follows:
https://github.com/hyperledger/indy-sdk/blob/master/libnullpay/src/lib.rs#L27

And the implementation of nullpay logger and posgresql seems to be currently quite different. 

Or are you just referring to way of error handling, eg libnullpay's
```
    if let Err(err) = utils::logger::LibnullpayLogger::init() {
        return err;
    }
```

[the remark this answers is not in the document]
Comment (Artemkaaas) [reply experimental/plugins/postgres_storage/src/lib.rs]: I mean piping logs by getting the current `log` function set in libindy and using it in plugin also.
it is a better way of gathering logs in my opinion. 
https://github.com/hyperledger/indy-sdk/blob/master/libnullpay/src/utils/logger.rs#L32

File: experimental/plugins/postgres_storage/src/lib.rs (hunk 1 above)
Comment (Patrik-Stas) [reply experimental/plugins/postgres_storage/src/lib.rs]: rewritten by example of libnullpay

File: experimental/plugins/postgres_storage/src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Patrik-Stas) [reply experimental/plugins/postgres_storage/src/lib.rs]: Makes sense, done!

**Pull request merged**